lib/capybara.rb

```diff
@@ -45,13 +45,55 @@ class << self
     end
 
     # Delegate Capybara global configurations
+    # @!method always_include_port
+    #   See {Capybara.configure}
+    # @!method run_server
+    #   See {Capybara.configure}
     # @!method default_selector
     #   See {Capybara.configure}
     # @!method default_max_wait_time
     #   See {Capybara.configure}
+    # @!method ignore_hidden_elements
+    #   See {Capybara.configure}
+    # @!method automatic_reload
+    #   See {Capybara.configure}
+    # @!method match
+    #   See {Capybara.configure}
+    # @!method exact
+    #   See {Capybara.configure}
+    # @!method exact_text
+    #   See {Capybara.configure}
+    # @!method raise_server_errors
+    #   See {Capybara.configure}
+    # @!method visible_text_only
+    #   See {Capybara.configure}
+    # @!method automatic_label_click
+    #   See {Capybara.configure}
+    # @!method enable_aria_label
+    #   See {Capybara.configure}
+    # @!method save_path
+    #   See {Capybara.configure}
+    # @!method asset_host
+    #   See {Capybara.configure}
+    # @!method default_host
+    #   See {Capybara.configure}
     # @!method app_host
     #   See {Capybara.configure}
-    # @!method always_include_port
+    # @!method server_host
+    #   See {Capybara.configure}
+    # @!method server_port
+    #   See {Capybara.configure}
+    # @!method server_errors
+    #   See {Capybara.configure}
+    # @!method default_set_options
+    #   See {Capybara.configure}
+    # @!method disable_animation
+    #   See {Capybara.configure}
+    # @!method test_id
+    #   See {Capybara.configure}
+    # @!method predicates_wait
+    #   See {Capybara.configure}
+    # @!method default_normalize_ws
     #   See {Capybara.configure}
     SessionConfig::OPTIONS.each do |method|
```

Comment:
Same order as defined in `SessionConfig`:

https://github.com/teamcapybara/capybara/blob/31e3c56b94cc2eef8778bad18f58ed2e6bf51648/lib/capybara/session/config.rb#L15-L63